packages/dev/core/src/Cameras/targetCamera.ts

```diff
@@ -119,6 +114,11 @@ export class TargetCamera extends Camera {
      */
     constructor(name: string, position: Vector3, scene?: Scene, setActiveOnSceneIfNoneActive = true) {
         super(name, position, scene, setActiveOnSceneIfNoneActive);
+
+        this._referencePoint = Vector3.Forward(this.getScene().useRightHandedSystem);
+
+        // Set the y component of the rotation to Math.PI in right-handed system for backwards compatibility.

```

Comment:
So this will keep the "wrong" behavior from before, but the math is now correct, is that right? So this means that left handed and right handed cameras will behave differently still?

Reply:
This isn't really wrong or right. The default behavior of a target camera in Babylon.js looks towards +Z in RH and LH. This just preserves that convention.